django_downloadview/api.py

```diff
@@ -20,3 +21,7 @@
 from django_downloadview.test import (assert_download_response,  # NoQA
                                       setup_view,
                                       temporary_media_root)
+
+
+# Backward compatibility.
+StringIteratorIO = TextIteratorIO
```

Comment:
StringIteratorIO is kept for backward compatibility, as an alias for TextIteratorIO.